operator/docs/user-guides/open-telemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "OpenTelemetry / OTLP"
+description: ""
+lead: ""
+date: 2024-10-25T12:43:23+02:00
+lastmod: 2024-10-25T12:43:23+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "user-guides"
+weight: 100
+toc: true
+---
+
+## Introduction
+
+Loki 3.0 introduced the OpenTelemetry Protocol (OTLP) as a new API for ingesting log entries in addition to the standard Push API that was available in previous versions of Loki.
+
+Because OTLP is not specifically geared towards Loki but is a standard format, it needs additional configuration on Loki's side to map the OpenTelemetry data format to Loki's data model.
+
+Specifically, OTLP has no concept of "stream labels" or "structured metadata". OTLP instead provides metadata about a log entry in _attributes_ that are grouped into three buckets (resource, scope and log), depending on the number of entries an attribute applies to.
+
+## Prerequisites
+
+Log ingestion using OTLP depends on structured metadata being available in Loki. This capability was introduced with schema version 13, which is available in Loki Operator when using `v13` in a schema configuration entry.
+
+When creating a new `LokiStack`, make sure to include `version: v13` in the storage schema configuration. If there is an existing schema configuration, a new configuration needs to be added, so that it becomes active in the future (see [Upgrading Schemas](loki-upgrading-schemas) in the Loki documentation).
+
+```yaml
+# [...]
+spec:
+  storage:
+    schemas:
+    - version: v13
+      effectiveDate: 2024-10-25
+```
+
+Once the `effectiveDate` has passed your `LokiStack` will be using the new schema configuration and is ready to store structured metadata.
+
+## Attribute Mapping
```

Comment:
Should we briefly describe the difference between stream label & structured metadata? In this section or other section that you find relevant

Reply:
I tried getting around that by linking to the stream-label / structured-metadata documentation on the Loki site. Another idea I had, was for us to have a few "concepts" pages where we can describe things like stream-labels, structured-metadata, schema versions and so on.

I'll think about this again. Maybe it makes sense to make this distinction clear in this document as well in a short paragraph.